pom.xml

```diff
@@ -168,6 +168,7 @@
     <maven-checkstyle-plugin.version>3.3.0</maven-checkstyle-plugin.version>
     <maven-shade-plugin.version>3.6.0</maven-shade-plugin.version>
     <mvel2.version>2.5.2.Final</mvel2.version>
+    <spotless.version>2.43.0</spotless.version>
```

Comment:
This might break for JDK 8. 😞 
```
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.43.0:apply (default-cli) on project phoenix: Execution default-cli of goal com.diffplug.spotless:spotless-maven-plugin:2.43.0:apply failed: Unable to load the mojo 'apply' in the plugin 'com.diffplug.spotless:spotless-maven-plugin:2.43.0' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: com/diffplug/spotless/maven/SpotlessApplyMojo has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

```

Reply:
Ref: https://github.com/diffplug/spotless/issues/1337